cmd/capabilities_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+// nolint: goconst // string duplication is for test readability.
```

Comment:
Is this needed?

Reply:
Good catch! 7d1e30cc60f2153d757c5e86bb6a73286eec79fa